tree/ntuple/v7/src/RField.cxx

```diff
@@ -1481,12 +1482,21 @@ ROOT::Experimental::RClassField::RClassField(std::string_view fieldName, std::st
 
       std::string typeName{GetNormalizedTypeName(dataMember->GetTrueTypeName())};
       std::string typeAlias{GetNormalizedTypeName(dataMember->GetFullTypeName())};
+
       // For C-style arrays, complete the type name with the size for each dimension, e.g. `int[4][2]`
       if (dataMember->Property() & kIsArray) {
          for (int dim = 0, n = dataMember->GetArrayDim(); dim < n; ++dim)
             typeName += "[" + std::to_string(dataMember->GetMaxIndex(dim)) + "]";
       }
-      auto subField = RFieldBase::Create(dataMember->GetName(), typeName, typeAlias).Unwrap();
+
+      std::unique_ptr<RFieldBase> subField;
+
+      const char *dmComment = dataMember->GetTitle();
+      if (dmComment && (strlen(dmComment) >= 2) && (strncmp("||", dmComment, 2) == 0)) {
```

Comment:
We also need to decide what to do with the case where the `TClass`/type of the data member as `TClass::CanSplit()` return false (typically because the class has a custom streamer that do things 'differently').

Reading the related `TStreamerElement::CannotSplit()`, I am reminded to ask the question, do we currently handle `TObject` correctly (i.e. either reject it or do the 'right' thing upon reading `fBits` and `fUniqueID`)?

Reply:
Two follow-up issues: #14808 and #14809